articles/howtos/create-new-decorator.md

```diff
@@ -0,0 +1,222 @@
+---
+uid: CreateNewDecorator
+---
+# How can I create a new Decorator?
+
+Click [here](xref:Decorator) to find out more about Decorators.
```

Comment:
I wonder if it's better to put this at the end as further reading, it will avoid any readers getting distracted for this article. However, if one doesn't know what a decorator is, then I suppose reading the article isn't going to be useful. I'm interested in other people's thoughts here, @joelsteventurner, @garethbaars?

Reply:
Why so @JonathanLydall ? Is that Decorator page not well written to explain that?

Reply:
My concern is that having the link at the start here _might_ make people feel they need to read more about decorators first, distracting them from the goal of the how-to. By moving it to the end of the article instead, it may be less distracting.

We may need to discuss as a team or find more opinion online.

Reply:
Or maybe we just add hyperlinks on "Decorator" keywords to take you to the reference page?
Also, I asked the question because you said "However, if one doesn't know what a decorator is, then I suppose reading the article isn't going to be useful.", so I wanted to know if it is because the reference page wasn't clear enough.

Reply:
Cool, we'll add a link on the line below this one.

```suggestion
```